rust/protocol/src/state/prekey.rs

```diff
@@ -7,7 +7,21 @@ use crate::proto::storage::PreKeyRecordStructure;
 use crate::{KeyPair, PrivateKey, PublicKey, Result};
 use prost::Message;
 
-pub type PreKeyId = u32;
+/// A unique identifier selecting among this client's known pre-keys.
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq, PartialOrd, Ord)]
+pub struct PreKeyId(pub u32);
```

Comment:
Same as DeviceId about `pub`, and also I think this one doesn't need Ord/PartialOrd.

Reply:
Done!